Package.swift

```diff
@@ -4,15 +4,14 @@ import PackageDescription
 let package = Package(
     name: "swift-otel-semantic-conventions",
     products: [
-        .library(name: "OTelSemanticConventions", targets: ["OTelHTTPTracingConventions"]),
         .library(name: "OTelSemanticMetricsConventions", targets: ["OTelHTTPMetricsConventions"]),
-        .library(name: "OTelSemanticTracingConventions", targets: ["OTelHTTPTracingConventions"]),
         .library(
             name: "OTelHTTPConventions",
-            targets: ["OTelHTTPMetricsConventions", "OTelHTTPTracingConventions"]
+            targets: ["OTelHTTPMetricsConventions"]
         ),
         .library(name: "OTelHTTPMetricsConventions", targets: ["OTelHTTPMetricsConventions"]),
-        .library(name: "OTelHTTPTracingConventions", targets: ["OTelHTTPTracingConventions"]),
+        .library(name: "OTelConventions", targets: ["OTelConventions"]),
```

Comment:
Open question: Should this be `OTelSemanticConventions` to match the repo name more closely?

Reply:
Sure, I had this same thought but was waiting for our decisions on namespace packages to shake out. I think I prefer `OTelSemanticConventions` over `OTelConventions` now. Do you think we should change to that?

Reply:
Yes, I'd change this to `OTelSemanticConventions` regardless of the traits discussion.

Reply:
Sounds good, I've changed it to `OTelSemanticConventions` in https://github.com/swift-otel/swift-otel-semantic-conventions/pull/2/commits/cec18486faea985619e84fbce772f7aa650c9f39